nuxeo-runtime/nuxeo-launcher-commons/src/test/resources/configurator/nuxeo.conf

```diff
@@ -21,3 +21,12 @@ nuxeo.wizard.done=false
 test.sampled.prop2=someValue
 
 nuxeo.force.generation=true
+
+nuxeo.db.host=${env:NUXEO_DB_HOST}
+
+org.nuxeo.fake.vindoz=${env?:NUXEO_UNKNOWN_VAR}
```

Comment:
:-1:  that syntax is useless: at the end, you create a String valuing "true" or "false", even not a boolean
and it is confusing: I tough initially that you it would mean "if env set, then set the var with the env value, else let the var unchanged"

What is the benefit of `org.nuxeo.fake.vindoz=${env?:NUXEO_UNKNOWN_VAR}` 
compared to something like `org.nuxeo.fake.vindoz=${env:NUXEO_UNKNOWN_VAR:false}` ?

Do you have a specific use case requiring a boolean string value based on the existence of an environment variable?